src/pages/ProductDetail/components/CountButton/CountButton.scss

```
@@ -0,0 +1,26 @@
+.count-button {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  width: 150px;
+  height: 70px;
+  margin: 10px;
+  border: 1px solid black;
+  border-radius: 40px;
+
+  .num {
+    font-size: 26px;
+  }
+
+  .button {
+    margin: 5px;
+    border: none;
+    background-color: rgba(1, 1, 1, 0);
+    color: rgb(211, 211, 211);
+    font-size: 40px;
+  }
+
+  button:hover {
+    color: black;
+  }
```

```suggestion
  .button {
    margin: 5px;
    border: none;
    background-color: rgba(1, 1, 1, 0);
    color: rgb(211, 211, 211);
    font-size: 40px;
    
      &:hover {
    color: black;
  }
  }
```

&연산자를 사용해서 스타일을 구현할 수 있겠습니다!